miden-lib/src/testing/mod.rs

```diff
@@ -167,12 +165,3 @@ pub fn mock_executed_tx(asset_preservation: AssetPreservationStatus) -> Executed
 
     ExecutedTransaction::new(program, tx_inputs, tx_outputs, account_delta, tx_args, advice_witness)
 }
-
-// HELPER FUNCTIONS
-// ================================================================================================
-
-fn build_dummy_tx_program() -> Program {
-    let operations = vec![Operation::Push(ZERO), Operation::Drop];
-    let span = miden_objects::vm::CodeBlock::new_span(operations);
-    Program::new(span)
-}
```

Comment:
I noticed this was moved to miden_objects, but considering this is a tx_program, wouldn't it also make sense to keep in `miden_tx`?

Reply:
Agreed, I had it in `miden-lib` at first but `Operation` was not available there, so I moved it down because `miden-lib` did not have `miden-tx` as a dependenciy. But since I added it as a dev dependency now, I think I can move it there.